pkg/master/storage_apps.go

```diff
@@ -0,0 +1,36 @@
+package master
+
+import (
+	"k8s.io/kubernetes/pkg/api/rest"
+	"k8s.io/kubernetes/pkg/apis/apps"
+	appsapiv1alpha1 "k8s.io/kubernetes/pkg/apis/apps/v1alpha1"
+	"k8s.io/kubernetes/pkg/genericapiserver"
+	petsetetcd "k8s.io/kubernetes/pkg/registry/petset/etcd"
+)
+
+type AppsRESTStorageProvider struct{}
+
+var _ RESTStorageProvider = &AppsRESTStorageProvider{}
+
+func (p AppsRESTStorageProvider) NewRESTStorage(apiResourceConfigSource genericapiserver.APIResourceConfigSource, restOptionsGetter RESTOptionsGetter) (genericapiserver.APIGroupInfo, bool) {
```

Comment:
how about some short go style names? e.g. rcs and g. Would make it much easier to read.